103-bundle-runtime.md

```diff
@@ -84,6 +84,59 @@ In addition to the default actions, CNAB runtimes MAY support custom actions (as
 
 A bundle MUST exit with an error if the action is executed, but fails to run to completion. A CNAB runtime MUST issue an error if a bundle issues an error. And an error MUST NOT be issued if one of the three built-in actions is requested, but not present in the bundle. Errors are reserved for cases where something has gone wrong.
 
+#### Well known custom actions
+
+An invocation image MAY implement any of the following well known custom actions. A CNAB indicates that it supports those actions by including them in its custom action list (as defined in [the bundle definition](101-bundle-json.md)).
+- `io.cnab.help` (with `stateless`: true and `modifies`: false): print an help message to the standard output. Implementations MAY print different messages depending on the parameters values passed to the invocation image.
+- `io.cnab.log` (with `stateless`: false and `modifies`: false): print logs of the installed system to the standard output.
+- `io.cnab.status` (with `stateless`: false and `modifies`: false): print a human readable status message to the standard output.
+- `io.cnab.status+json` (with `stateless`: false and `modifies`: false): print a json payload describing the detailed status with the following schema:
+
+```json
+{
+    "status": "Failed",
+    "message": "Component frontend failed to deploy properly",
```

Comment:
Nit, the component is called `front-end` below. I suppose an implementation is at liberty to be inconsistent but maybe not to be encouraged.